### PR TITLE
[HAC-998] Deploy to Ephemeral

### PR DESCRIPTION
**Pull request by BlakeHolifield**
* Deploys HAC and its dependences to a random ephemeral environment
* Sets up authentication in the central keycloak
* Registers the service for use in the dev sandbox cluster.

On every PR that's opened in this repo, it will deploy a frontend for testing. e2e tests with Cypress will come as next steps.

### CODE REVIEW

File: tmp/keycloak.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        "attributes": {
+            "firstName": "Ephemeral",
+            "lastName": "Environment",
+            "AccountID": "12345",
+            "AccountNumber": 12345,
+            "OrgID": "12345",
+            "IsInternal": "false",
+            "IsOrgAdmin": "true",
+            "IsActive": "true",
+            "Entitlements": '"insights": {"is_entitled": "true", "is_trial": "false"}'
```
Comment (karelhala) [inline tmp/keycloak.py:57]: Do we need to send `insights` entitlements? I think it should be enough to just send empty object.

Comment (BlakeHolifield) [reply tmp/keycloak.py:57]: I talked with Pete, and supporting this is going to be a bit of an effort to update hac-proxy. The user we're adding is already registered and we'd like to get this merged for e2e support.

File: tmp/hac-proxy.yaml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      rules:
+        - host: ${HOSTNAME}
+          http:
+            paths:
+              - backend:
+                  service:
+                    name: hac-proxy
+                    port:
+                      number: 8080
+                path: /api/k8s
```
[the remark this answers is not in the document]
Comment (BlakeHolifield) [reply tmp/hac-proxy.yaml]: I talked with Pete, and supporting this is going to be a bit of an effort to update hac-proxy. The user we're adding is already registered and we'd like to get this merged for e2e support.

**Review by psav — APPROVED** [review]

**Review by jerolimov — APPROVED** [review]
Thanks Pete for confirm this change.

/approve

**Pull request merged**